src/components/CheckoutTabs/CheckoutOrderTab.tsx

```diff
@@ -17,14 +17,14 @@ export const CheckoutOrderTab = (): ReactElement => {
   const orderExists = localOrder.lineItems?.length > 0;
 
   return (
-    <div className="flex h-full w-full flex-col">
+    <div className="flex min-h-[65vh] w-full flex-col">
       <div className="w-full">
-        <h1 className="text-ak-grey-1 mb-4 text-[20px] font-[600]">Order</h1>
+        <h1 className="text-ak-grey-1 m-1 text-[20px] font-[600]">Order</h1>
       </div>
 
       {!orderExists ? (
         <ComponentTransition lineItemsExist={orderExists}>
-          <div className="text-ak-grey-1 mx-auto translate-y-[50%] items-center justify-center text-center text-[14px] font-[400]">
+          <div className="text-ak-grey-1 mx-auto translate-y-[70%] items-center justify-center text-center text-[14px] font-[400]">
```

Comment:
now that we have set the height of the parent, could you try centring the image in a better way than I did previously. Translate-y-etc is a bit hacky